### PR TITLE
Add compressed js file to be ignored from Transifex

### LINKED ISSUE #247: Text strings  in Transifex
There are strings from `workflow_pdf_converter-main.js` in Transifex that should not appear for translation.

### PR DESCRIPTION
**Pull request by rakekniven**
Fixes #247 

Signed-off-by: rakekniven <mark.ziegler@rakekniven.de>

### CODE REVIEW

**Comment by rakekniven** [comment]
Please merge. It is urgent, nearly 2000 Strings are affected.

File: .l10nignore [hunk 1]
```diff
@@ -1 +1,2 @@
 js/workflow_pdf_converter.js
+js/savedworkflow_pdf_converter-main.js
```
Comment (blizzz) [inline .l10nignore:2]: the path looks suspicious

```suggestion
js/workflow_pdf_converter-main.js 
```

**Comment by rakekniven** [comment]
Hello @blizzz 
Damn copy and paste. Good to have attentive reviewers.

**Review by blizzz — APPROVED** [review]

**Comment by rakekniven** [comment]
I know that what are reviewers for. Thank you anyway.

**Comment by blizzz** [comment]
Thank you @rakekniven!

Yeah, little mistakes like to sneak in occasionally. No worries :)

**Pull request merged**